legacy/definition.go

```diff
@@ -537,6 +537,7 @@ func migrateAction(baseLanguage utils.Language, a Action, localization flows.Loc
 
 		return &actions.RemoveContactGroupsAction{
 			Groups:     groups,
+			AllGroups:  len(groups) == 0,
```

Comment:
so old flows represented this as an empty list then?

Reply:
Yup, checked it